Multisig/UI/App/ContentView.swift

```diff
@@ -30,7 +30,7 @@ struct ContentView: View {
 
                 BottomTabView($viewState.state) {
 
-                    TempAssets()
+                    AssetsView()
```

Comment:
Please rename this tab to the "Assets" as it is now in Zeplin

Reply:
ok